src/actions/index.js

```diff
@@ -0,0 +1,32 @@
+export const DELETE_CARD = 'DELETE_CARD';
+export const SHOW_ADD_MODAL = 'SHOW_ADD_MODAL';
+export const HIDE_ADD_MODAL = 'HIDE_ADD_MODAL';
+export const ADD_CARD = 'ADD_CARD';
+
+export const deleteCard = key => {
+  return {
+    type: DELETE_CARD,
+    key: key
+  };
+};
+
+export const showAddModal = () => {
+  return {
+    type: SHOW_ADD_MODAL,
+    display: 'block'
```

Comment:
I think in general for our actions we wouldn't pass CSS properties rather than passing data through the action call to the reducer as you did w/ `addCard`. It's really up to the UI dependency of the store to determine what to do based on a state of data than the action telling the UI what to do as seen here in the modal handler.